src/test/ui/wf/wf-unsafe-trait-obj-match.stderr

```diff
@@ -25,6 +28,9 @@ LL |         Some(()) => &S,
 error[E0038]: the trait `Trait` cannot be made into an object
   --> $DIR/wf-unsafe-trait-obj-match.rs:25:25
    |
+LL | trait Trait: Sized {}
+   |              ----- the trait cannot require that `Self : Sized`
```

Comment:
so this is a bit confusing. A trait can very well require `Self: Sized`, but then you can't turn it into a `dyn Trait` anymore

Reply:
I didn't modify the pre-existing wording. Do you have some suggestions?

Reply:
Changed it to

```
error[E0038]: the trait `Trait` cannot be made into an object
  --> $DIR/wf-unsafe-trait-obj-match.rs:25:25
   |
LL | trait Trait: Sized {}
   |              ----- traits that require `Self: Sized` cannot be made into an object
...
LL |     let t: &dyn Trait = match opt() {
   |                         ^^^^^^^^^^^ the trait `Trait` cannot be made into an object
   |
   = note: required because of the requirements on the impl of `std::ops::CoerceUnsized<&dyn Trait>` for `&R`
   = note: required by cast to type `&dyn Trait`
```

which is hopefully clearer.